### PR TITLE
Better terminal output when using the scaffolding app

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by helfi92** [comment]
When running the scaffolding app, users may have the tendency to run `yarn` or `npm install` after changing into the directory when in fact, it is not needed. This pull-requests makes it explicit, hopefully making people happier and less confused :)

**Comment by helfi92** [comment]
screen shot 2017-11-17 at 10 01 08 am

File: packages/create-neutrino-project/commands/init/index.js [hunk 1]
```diff
@@ -181,7 +191,7 @@ module.exports = class Project extends Generator {
   }
 
   end() {
-    this.log(`Success! Created ${this.options.directory} at ${process.cwd()}`);
+    this.log(`${chalk.green('Success')}! Created ${this.options.directory} at ${process.cwd()}`);
```
Comment (eliperelman) [inline packages/create-neutrino-project/commands/init/index.js]: Put the exclamation point inside the green too. :)

**Comment by helfi92** [comment]
Pushed a commit to it then rebased.

**Pull request merged**